packages/typespec-ts/test/modularIntegration/generated/models/propertyAdditional/src/rest/outputModels.ts

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/** The model extends from Record<unknown> type. */
+export interface ExtendsUnknownAdditionalPropertiesOutput
+  extends Record<string, any> {
```

Comment:
Considering the type generated in rlc and modular is different, not sure there would happen for compile error between them because of type inconsistancy?

Reply:
Can you point out what is the difference here? the legacy client should be the same in both Modular and RLC.

Reply:
or you mean unknown specifically? which we have to use any for both input and output as we don't split the input models and output models in Modular.

Reply:
Not sure if @joheredi has any different thoughts.

Reply:
Thanks for the explainantion! `any` makes sense for me. I just was thinking because there exist type difference between rlc and modular, for example tsp datetime would be translated to Date in modular but string in RLC, would it be a problem here?

Another question is if there exist ser/derser logic e.g properties renaming, how to handle additional properties? is it in current pr scope?